test-apps/imodel-transformer/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "imodel-transformer",
+  "//private": "NOTE: this is a private package that is never published",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc 1>&2",
```

Comment:
Can you add a `compile` that just calls `build`?

Reply:
Done